core/math/random.h

```diff
@@ -25,6 +25,7 @@ class Random : public RandomNumberGenerator {
 
 	Variant range(const Variant &p_from, const Variant &p_to);
 	Variant choice(const Variant &p_sequence);
+	Variant choices(const Variant &p_sequence, int count, const Array &p_weights, bool is_cumulative);
```

Comment:
Prepend `p_` for parameters (Godot naming convention).

Also, given that the code could be used in C++ as well, it's better to provide default arguments in the method signature.

The following suggestion applies both ideas:

```suggestion
	Variant choices(const Variant &p_sequence, int p_count = 1, const Array &p_weights = Variant(), bool p_cumulative = false);
```

Don't forget to do the same to `cpp` file.